src/applications/facility-locator/containers/VAMap.jsx

```diff
@@ -559,7 +559,10 @@ class VAMap extends Component {
         </div>
 
         <div className="facility-introtext">
-          Find VA locations near you with our facility locator tool. You can search for your nearest VA medical center as well as other health facilities, benefit offices, cemeteries, and Vet Centers. You can also filter your results by service type to find locations that offer the specific service you’re looking for.
+          Find VA locations near you with our facility locator tool. You can search for your nearest 
+          VA medical center as well as other health facilities, benefit offices, cemeteries, community 
```

Comment:
Should this go behind a flag for now?

Reply:
Yea. Yea it probably should. 👌 
Think it would be better to have two different blocks and wrap them in a feature flag to pick the one to show? Or just put an inline flag around the new text?
e.g.
```jsx
<div className="facility-introtext">
  Find VA locations near you with our facility locator tool. You can search for your nearest 
  VA medical center as well as other health facilities, benefit offices, cemeteries, 
  { ccLocatorEnabled() && <Fragment>community care providers, </Fragment> }
  and Vet Centers. You can also filter your results by service type to find 
  locations that offer the specific service you’re looking for.
</div>
```

Reply:
I don't think we can use Fragments yet, but typically we favor whatever is easier to delete later.

Reply:
Ah, thought there was a way to get by with `<React.Fragment>`. Ended up going with a `<span>` for now. Should be updated like above now.